src/main/java/graphql/nadel/Nadel.java

```diff
@@ -224,4 +212,54 @@ private InvalidDslException hydrationError(InnerServiceHydration hydration, Stri
         return this.braid.newGraphQL().execute(executionInput);
     }
 
+    /**
+     * @return a builder of Nadel objects
+     */
+    public static Builder newNadel() {
+        return new Builder();
+    }
+
+    public static class Builder {
+        String dsl;
```

Comment:
Do you think we could reduce these variables accessibility to private final?

Reply:
we can make them private but not final.  Its a Builder after all and hence mutable by design.

For the record the Builder pattern with a static `newFoo` pattern has served us really well in graphql-java.  Stops the 6 constructors pattern we had above

Reply:
fixed